browser/calculator.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title></title>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css" />
```

Comment:
You should include this file from a CDN (here's a link: https://cdnjs.com/libraries/normalize).  This reduces the load on your server when your awesome new calculator is deployed to production, and gets the content to the client faster.